### DISCUSSION TITLE
Relative link to another page in Wiki results in <image/> tag?

### DISCUSSION THREAD

**Discussion by ero-qt** [community/community#109492] [Repositories]
### Select Topic Area

Bug

### Body

The raw contents of the wiki page at <https://github.com/just-ero/asl-help-v2/wiki/AslHelp.Memory-Namespace> is the following:
```md
## Classes

<table>
<tr>
  <td>[[Module|Module Class]]</td>
  <td>Represents an assembly loaded into a process.</td>
</tr>
</table>
```

However, for some reason, the `<td>[[Module|Module Class]]</td>` is converted into
```md
<td><img src="/just-ero/asl-help-v2/wiki/AslHelp.Memory/Module"></td>
```

I have no clue why this would happen, especially because it does not happen on other pages.

**Comment by its21yash** [comment]
It seems like there might be an issue with the syntax you're using in the wiki page. The conversion of `<td>[[Module|Module Class]]</td>` into `<td><img src="/just-ero/asl-help-v2/wiki/AslHelp.Memory/Module"></td>` suggests that GitHub might be interpreting it as an image link instead of a regular link.

One possible solution is to use the markdown link syntax directly, without enclosing it in HTML `<td>` tags. Try rewriting the table row like this:

`| [[Module|Module Class]] | Represents an assembly loaded into a process. |`

This way, GitHub will interpret the link correctly as a markdown link and display it properly in the wiki page.

If the issue persists, there might be some specific formatting or configuration causing this behavior, and you may need to reach out to GitHub support for further assistance.

**Reply by ero-qt** [reply]
The same issue persists (you can open the same page again). Strange.

The reason I don't go with a markdown table is because GitHub-flavored markdown does not allow for header-less tables.

**Comment by ero-qt** [comment]
I have noticed that this does *not* happen if I move the wiki page file into another directory. Behind the scenes (in the `.wiki` repo), the file structure is the below:

![image](https://github.com/community/community/assets/56401411/e15b2410-7619-4b92-9a12-094e243dc617)

When I move `AslHelp.Memory Namespace.md` to the root, or a directory of its own, this issue goes away.

**Comment by MasterInQuestion** [comment]
Use ["&lt;a>"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a).

**Reply by ero-qt** [reply]
I wasn't really looking for a solution, since solving this is trivial (as you have shown). This is more of a bug report.

Additionally, an anchor tag doesn't fulfill the same behavior as a wiki link (shows up red if the page does not exist, otherwise blue).

**Reply by MasterInQuestion** [reply]
Thanks for the notice.

    In addition I noticed the link syntax used doesn't look like GFM, but sort of MediaWiki Wikitext derived?
    Try &#x60;`[[Module|:Module Class]]`&#x60;?

    See:
    https://www.mediawiki.org/wiki/Help:Links (":File")
    https://en.wikipedia.org/wiki/Help:Link

**Reply by ero-qt** [reply]
>  Try `[[Module|:Module Class]]`?

Interestingly, this does not turn the link into an `<image>`. But it now (for obvious reasons) links to `wiki/%3AModule-Class`, which is a page that doesn't exist. The link is thus red.

**Reply by MasterInQuestion** [reply]
Does using the Markdown style &#x60;`[Text](URI)`&#x60; work?

**Reply by ero-qt** [reply]
> Does using the Markdown style `[Text](URI)` work?

Certainly, but once again, that's not a wiki link. It does not end up red when the target page does not exist.
